### ISSUE TITLE
Errors during unit tests

### ISSUE THREAD

**Issue by pvcraven** [pythonarcade/arcade#805]
The test_drawing_dimensions and test_physics_engine_platformer occasionally are generating errors like this:
```
FAILED                       [ 58%]
tests\unit2\test_drawing_dimensions.py:73 (test_main)
def test_main():
        """ Main method """
        window = MyGame(SCREEN_WIDTH, SCREEN_HEIGHT)
>       window.test()

C:\Users\craven\Desktop\WebServer\arcade\tests\unit2\test_drawing_dimensions.py:77: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
C:\Users\craven\Desktop\WebServer\arcade\arcade\application.py:454: in test
    self.dispatch_events()
C:\Users\craven\Desktop\WebServer\arcade\arcade\application.py:569: in dispatch_events
    super().dispatch_events()
C:\Users\craven\Desktop\WebServer\arcade\venv38\lib\site-packages\pyglet\window\win32\__init__.py:688: in dispatch_events
    self.dispatch_pending_events()
C:\Users\craven\Desktop\WebServer\arcade\venv38\lib\site-packages\pyglet\window\win32\__init__.py:701: in dispatch_pending_events
    EventDispatcher.dispatch_event(self, *event)
C:\Users\craven\Desktop\WebServer\arcade\venv38\lib\site-packages\pyglet\event.py:415: in dispatch_event
    if getattr(self, event_type)(*args):
C:\Users\craven\Desktop\WebServer\arcade\tests\unit2\test_drawing_dimensions.py:46: in on_draw
    arcade.draw_arc_outline(x, y, width, 50, arcade.color.AFRICAN_VIOLET, 0, 180, 5)
C:\Users\craven\Desktop\WebServer\arcade\arcade\draw_commands.py:142: in draw_arc_outline
    _generic_draw_line_strip(point_list, color, gl.GL_TRIANGLE_STRIP)
C:\Users\craven\Desktop\WebServer\arcade\arcade\draw_commands.py:460: in _generic_draw_line_strip
    geometry.render(program, mode=mode)
C:\Users\craven\Desktop\WebServer\arcade\arcade\gl\vertex_array.py:390: in render
    vao.render(
C:\Users\craven\Desktop\WebServer\arcade\arcade\gl\vertex_array.py:199: in render
    gl.glBindVertexArray(self.glo)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

result = None, func = <WinFunctionType object at 0x1458D3A8>
arguments = (c_ulong(1),)

    def errcheck(result, func, arguments):
        if _debug_gl_trace:
            try:
                name = func.__name__
            except AttributeError:
                name = repr(func)
            if _debug_gl_trace_args:
                trace_args = ', '.join([repr(arg)[:20] for arg in arguments])
                print('%s(%s)' % (name, trace_args))
            else:
                print(name)
    
        from pyglet import gl
        context = gl.current_context
        if not context:
            raise GLException('No GL context; create a Window first')
        if not context._gl_begin:
            error = gl.glGetError()
            if error:
                msg = ctypes.cast(gl.gluErrorString(error), ctypes.c_char_p).value
>               raise GLException(msg)
E               pyglet.gl.lib.GLException: b'invalid operation'

C:\Users\craven\Desktop\WebServer\arcade\venv38\lib\site-packages\pyglet\gl\lib.py:107: GLException

```

Run individually, these tests do not generate errors.

**Comment by pvcraven** [comment]
Both have tests right before that import arcade, but do not open a window. I was able to get the draw ellipse test to pass by disabling the test before it, but not the physics engine one.

**Comment by pvcraven** [comment]
Fix shipping with 2.5.1
